nltk/parse/corenlp.py

```diff
@@ -339,9 +339,11 @@ def tag_sents(self, sentences):
         """
         # Converting list(list(str)) -> list(str)
         sentences = (" ".join(words) for words in sentences)
-        return [sentences[0] for sentences in self.raw_tag_sents(sentences)]
+        if properties is None:
```

Comment:
Please add a description for the `properties` argument in the docstring, also mention that `"tokenize.whitespace"` is set to `True` by default (but it won't if `properties` are specified.